### ISSUE TITLE
Login / Lock screen improvements

### ISSUE THREAD

**Issue by Feichtmeier** [ubuntu/yaru#622]
This is the continuation of this Issue on the shell's old repo https://github.com/ubuntu/gnome-shell-communitheme/issues/183 about how we can improve the GNOME Shell's login/lock screen - knowing that it might get replaced in the future.

Solutions discussed so far:
- no gradient
- dark box 
- light box

**Comment by Feichtmeier** [comment]
Does everyone feel comfortable with that new login/lookscreen ?
Shamelessly tagging 
@didrocks @luxamman @madsrh @clobrano @godlyranchdressing @galgalesh 
If yes we could close this issue :)

**Comment by didrocks** [comment]
TBH, I'm not a big fan of it compared to the previous one. Unsure if I'll get used to it or not.

**Comment by Paz-it** [comment]
Although not tagged , I'll be happy to go back to previous one. TBH, I was already digging @didrocks  blog for how to change the background if the one is in use remain. :rofl:

**Comment by Feichtmeier** [comment]
Alright let's wait a bit for the missing thumbs down or up for some weeks and then roll back eventually.

**Comment by eaglersdeveloper** [comment]
The "Cancel" button is less noticeable.
Make the outline darker or add a shadow.

**Comment by eaglersdeveloper** [comment]
Is it possible to use the current wallpaper as a login background?
I would like to return to the previous one, but with wallpaper.
As in Windows 10 or macOS.

**Comment by clobrano** [comment]
Nope, not supported now

**Comment by Feichtmeier** [comment]
The cancel button needs some love indeed :)

**Comment by Feichtmeier** [comment]
![screenshot from 2018-07-25 17-18-58](https://user-images.githubusercontent.com/15329494/43210336-17b92c54-902f-11e8-8e19-0b95cf837837.png)
Like this, @eaglersdeveloper ?

**Comment by eaglersdeveloper** [comment]
@Feichtmeier, yes!

But `font-weight` of user name need set to `normal`.
And avatar. In Ubuntu Touch it was in Suru style, but with a 1:1 aspect ratio.

![2018-07-25 20-31-29](https://user-images.githubusercontent.com/40228953/43214366-11cc4d2e-904a-11e8-93b9-a2634b4ec619.png)

**Comment by Feichtmeier** [comment]
![screenshot from 2018-07-25 17-18-58](https://user-images.githubusercontent.com/15329494/43218429-b05083c6-9044-11e8-91dd-c5c230d3b861.png)

- normal font weight
- 1px less for the user picture border

**Comment by godlyranchdressing** [comment]
@Feichtmeier 

How about this?

![lock](https://user-images.githubusercontent.com/27529229/43297349-db0dc8a0-911e-11e8-8577-66a0bbc8204f.png)

Got it from here:
![dialog window](https://user-images.githubusercontent.com/27529229/43297355-e391c3dc-911e-11e8-8d3e-5e2e6a006aff.png)

But I'd get if that were too drastic.

![dialog](https://user-images.githubusercontent.com/27529229/43297383-052d9cfa-911f-11e8-80b8-d49802809200.png)

```
$light_borders_color: #8E8E8E;
$light_button_bg_color: #B1B1B1;
```

**Comment by clobrano** [comment]
I'd prefer current + [this](https://github.com/ubuntu/yaru/issues/622#issuecomment-407841634) for login

**Comment by Paz-it** [comment]
If keeping the current I'm +1 with @clobrano .

**Comment by eaglersdeveloper** [comment]
Is the current one without dialog?

**Comment by Feichtmeier** [comment]
![screenshot from 2018-07-27 21-39-50](https://user-images.githubusercontent.com/15329494/43343208-b506dd78-91e5-11e8-8655-235b7238f6b6.png)

Updated the "login with another user" label to have a normal font-weight, too.
:thinking: this looks very unity8ish and very "modern" :dancer:

**Comment by clobrano** [comment]
It woule be great if we could mimic some glass effect with gradient :angel:

**Comment by Feichtmeier** [comment]
![screenshot from 2018-07-28 11-14-41](https://user-images.githubusercontent.com/15329494/43355129-7e5cdb16-9257-11e8-8f43-71f47056b650.png)

With that gradients, the previously reported rendering problem with the SVG is basically gone. :man_shrugging:

**Comment by clobrano** [comment]
Awesome! Can we try some different gradients and tones? Someone suggested that previous gradient starting from the upper left corner make the eyes look on the wrong part of the screen, so I wonder if a radial gradient from the center might look better

**Comment by Feichtmeier** [comment]
Tones - yes! 
But with that undocumented gradients, only vertical and horizontal seems to work :(

**Comment by a-hayasi** [comment]
Hi! (This is my first post in any forum, sorry if I say something wrong). What do you think about the "Cancel" button with a red color?

**Comment by madsrh** [comment]
Hi @a-hayasi 

Red is recommended for negative and irreversible action buttons, errors and alerts.

But thanks for taking the time to contribute 👍 Good first post!

**Comment by clobrano** [comment]
closed by #661
